lib/grape-swagger/endpoint.rb

```diff
@@ -198,7 +198,7 @@ def response_object(route)
         response_model = expose_params_from_model(value[:model]) if value[:model]
 
         if route.request_method == 'DELETE' && !value[:model].nil?
```

Comment:
yeap, think for the moment – support grape < 0.19.0 – it is the best, the default of 204 for DELETE was introduced in 0.19.0